app/reports/hbx_reports/shop_enrollment_report.rb

```diff
@@ -64,7 +66,8 @@ def migrate
             subscriber_ssn = person.ssn
           end
           in_glue = glue_list.include?(id) if glue_list.present?
-          csv << [employer_id,fein,legal_name,plan_year_start,plan_year_state,employer_profile_aasm,eg_id,purchase_time,coverage_start,enrollment_state,subscriber_hbx_id,first_name,last_name,subscriber_ssn,plan_hios_id,covered_lives,enrollment_reason,in_glue]
+          csv << [employer_id,fein,legal_name,rating_area,plan_year_start,plan_year_state,employer_profile_aasm,eg_id,purchase_time,coverage_start,enrollment_state,subscriber_hbx_id,first_name,last_name,subscriber_ssn,plan_hios_id,is_pvp,
```

Comment:
It looks like rating area is in the 'Employer Plan Year Start Date' column, unless i'm confused. is that correct?